QUEST_LV_0100.tsv

```diff
@@ -287,17 +287,17 @@ QUEST_LV_0100_20150317_000286	$Shouldn't a Revelator be heading to Klaipeda inst
 QUEST_LV_0100_20150317_000287	$I feel so relieved to be stationed in the Western Woods.{nl}There are some places where you'd rather die than be stationed.
 QUEST_LV_0100_20150317_000288	$When you see a Goddess Statue, try offering your worship.{nl}Surely something nice will happen.
 QUEST_LV_0100_20150317_000289	$This is a dangerous area. What are you doing here?
-QUEST_LV_0100_20150317_000290	$Thank you.{nl}I better go report to Titas about the roaming Large Kepa.
+QUEST_LV_0100_20150317_000290	$Thank you.{nl}I should report the roaming Large Kepa to Titas.
```

Comment:
I should report the roaming Large Kepa -> I should report about the roaming Large Kepa

?